packages/protocol/contracts/common/UsingPrecompiles.sol

```diff
@@ -54,10 +59,45 @@ contract UsingPrecompiles {
           revert(returnDataPosition, returnDataSize)
         }
         default {
-          returnNumerator := mload(returnDataPosition)
-          returnDenominator := mload(add(returnDataPosition, 32))
+          switch returnDataSize
+            case 0 {
+              error := 1
+            }
+            default {
+              returnNumerator := mload(returnDataPosition)
+              returnDenominator := mload(add(returnDataPosition, 32))
+            }
         }
     }
+    if (error == 1) {
```

Comment:
Why is this necessary?

Reply:
oops, got some error and didn't realize the precompile was implemented in ganache